File: app/src/main/java/com/prideven/android/HungryEats/EatsRestaurantWebService.java
package com.prideven.android.hungryeats;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Header;
import retrofit2.http.Path;
import retrofit2.http.Query;

public interface EatsRestaurantWebService {

    @GET("v2/restaurant/")
    Call<EatsRestaurantsResponse> getRestaurants(@Query("lat") Double latitude, @Query("lng") Double longitude);
}

File: app/src/main/java/com/prideven/android/HungryEats/EatsRestaurantWebService.kt
package com.prideven.android.hungryeats

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface EatsRestaurantWebService {
    @GET("v2/restaurant/")
    fun getRestaurants(
        @Query("lat") latitude: Double?,
        @Query("lng") longitude: Double?
    ): Call<EatsRestaurantsResponse?>?
}